client/scripts/script.js

```diff
@@ -0,0 +1,31 @@
+/*
+$(document).ready(function() {
+	alert("jquery loaded");
+}); */
+
```

Comment:
This jquery looks great! Functions as expected. Common convention for jquery is we wrap all our code in the document ready selector. I believe this prevents our code to be run before we load the page.

```suggestion
$(document).ready(function() {
	getloginvalues();
	getsignupvalues(); 
}); 
```
This small fix should work.